packages/api/src/routes/internal.ts

```diff
@@ -343,6 +345,9 @@ router.get(
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getUUIDFrom("query", req, "cxId").orFail();
     const source = getFrom("query").optional("source", req);
+    if (source !== undefined && !isCxMappingSource(source)) {
+      throw new BadRequestError(`Invalid source ${source}`);
```

Comment:
nit: keep error messages static so they're grouped on Sentry. I think we could send `source` as `additionalInfo` and that should be returned - or at least sent to Sentry as well

Reply:
Done.